### DISCUSSION TITLE
Update Spotify FAQ and/or manual

### DISCUSSION THREAD

**Discussion by Micharius** [MiczFlor/RPi-Jukebox-RFID#1758] [Ideas]
Hi there, 
a couple of month ago, I created a Phoniebox for my daughter. I had quite a lot of trouble to get it running, mainly because Spotify did not work first. I tried multiple things, reinstalls and options, and finally I managed to get it working, altough I did not really know why it finally worked.
For a friend, I recently installed a second Phoniebox as a fresh installation. I know realized, what caused my so much trouble in my attempts to get the box running: If you do a fresh Spotify-Installation, and you want to use to Box as a Spotify-only device, after installation, the box will not play any Spotify sound. 
However, if you upload just a simple mp3 on the box, link it to a card and play it once, Spotify will work fine from that moment. I even did a second attempt to install the box in order to reproduce this issue. Somehow, if you just start trying to play Spotify right away, it is not possible to play any tracks. In my first attempts on the first box, I did several tests with the box, also with generic sound files on the box, but it took obviously some to time until I did the right order: Play a generic sound file from a card, THEN use Spotify. The fact, that I startet to dive deep into config files and different options problably made this process more complicated.
To be honest, I'm not sure if this fact is already known or maybe already mentioned in the Wiki, but I thought I point it out here. If not, it would be helpful for future users to mention this in the Wiki or Spotify FAQ.

Kind regards,

Micharius

**Comment by pabera** [comment]
Hey Micharius.. Thanks for pointing this out. I linked your post into the wiki so others can read about. Because you are describing a problem in version 2.x, I don't think this will actually be solved despite the fact that it is an unpleasant problem. Thanks for sharing and we hope to avoid this problem for 3.x ;-)

https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Spotify-FAQ

**Comment by harrydoddnoble** [comment]
Hi @Micharius  - I think I have this problem but is it too late to do it if you add a Spotify link as your first track? Do I need to do a clean install do you think?

**Comment by Micharius** [comment]
Hello, no, with the version I had when I wrote this post, I just had to play a local file from a card **once**, and Spotify right away worked fine. It was necessary.to do it as the absolute first thing after installation.

**Comment by Te-eMster** [comment]
I tried reproducing your steps and played a local mp3 file from an RFID-Card on a completely fresh install.

However, even after that any Spotify playlist will just stay silent plus autoplay for the other RFID-Card seems broken. Do you have any suggestions as to what might have failed?

**Comment by Micharius** [comment]
Hmm...sounds unusual...do you hear the Phoniebox startup single when it starts up? Because for me, that was the weird thing: I heard the sound, so clearly Phoniebox was wired correctly, but still no Spotifiy. How about yours?

**Reply by Te-eMster** [reply]
It was actually my bad, I was not aware that the bug isn't fixed with version 2.7, there was only a dev build that has the fix. 
So I had to manually perform the steps provided in https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2429#issuecomment-2445301802 . 
Now everything works fine :-)
As mentioned here  https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2418#issuecomment-2466913155  this will be fixed in the 2.8 release. I was just a few hours too early to install.
